app/Importer/Type.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Importer;
+
+enum Type: string
```

Comment:
I'm fine with this being an enum, but I don't like the `Type` name here - it's just going to be too confusing. Can we rename it to `ImportType` or something like that? It can continue to live here - that's fine, since it really does only have to do with importing - but if you wanted to put it into `App\Enums` that would be OK too.